### PR TITLE
Emit metadata for column name case-sensitivity

### PR DESCRIPTION
**Pull request by mihaibudiu**

### CODE REVIEW

**Comment by mihaibudiu** [comment]
Fixes #709

File: sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java [hunk 1]
```diff
@@ -132,6 +132,7 @@ public JsonNode getDefinedObjectSchema() {
         for (RelColumnMetadata col: this.columns) {
             ObjectNode column = fields.addObject();
             column.put("name", col.getName());
+            column.put("case_sensitive", col.nameIsQuoted);
```
Comment (ryzhyk) [inline sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java:135]: Will this work correctly for outputs?

Comment (mihaibudiu) [reply sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java:135]: Let me add a test where the users explicitly specify the column names in the view. We don't have any of these yet.
I think the question only makes sense for that case.
And the answer is probably "no".

Comment (ryzhyk) [reply sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java:135]: > I think the question only makes sense for that case.

Right, but also we want to make sure to output `case_sensitive=false` for columns with auto-generated names.

**Review by ryzhyk — APPROVED** [review]

**Pull request merged**